### ISSUE TITLE
git fetch is using "hard coded" plink.exe location

### ISSUE THREAD

**Issue by Muchaszewski** [gitextensions/gitextensions#9545]
## Current behaviour

git fetch is using "hard coded" plink.exe location

```
"git" fetch --progress "origin"
error: cannot spawn C:\Program Files (x86)\PuTTY\plink.exe: No such file or directory
fatal: unable to fork
Done

Press Enter or Esc to exit...
```
My current settings:

![image](https://user-images.githubusercontent.com/5092066/131346411-aa42ed13-ab25-48d7-a93a-2fde219a1794.png)

## Expected behaviour
git fetch should use path from SSH settings page in this case C:\Program Files\PuTTY\plink.exe

## Steps to reproduce
1 . Install putty in a different location then `C:\Program Files (x86)\PuTTY\plink.exe`
2. Try fetch using SSH

## Screenshots

![image](https://user-images.githubusercontent.com/5092066/131346661-d9fcffcb-3c65-4f0f-af86-86d574fcaa0b.png)

## Did this work in previous version of GitExtensions

Worked in the last version v3.5.1 and v3.5.2

## Environment

- GitExtensions version: v3.5.3
- GIT version: git version 2.31.1.windows.1
- OS version: Windows 10 x64
- .NET version: All required

## Diagnostics

[gitex.log](https://github.com/gitextensions/gitextensions/files/7077037/gitex.log)

**Comment by Muchaszewski** [comment]
I will not close this ticket as this is clearly an issue, but I managed to fix it.
I needed to change to a different option in SSH settings and bring back Putty and press apply.

https://user-images.githubusercontent.com/5092066/131348594-1e2db1af-8124-4b58-941f-1910e94ec3c4.mp4

This issue is clearly somewhere a default value being used after update of the executable and settings are not applied properly or read from properly, only after forced "refresh" of said settings it works.

**Comment by vbjay** [comment]
Correct. The installer blew away those settings. All you needed to do is
reset Tre paths to correct paths. Annoying I know.

**Comment by CookiePLMonster** [comment]
Related to/duplicate of #4011.
